pkg/updater/data/catalog.json

```diff
@@ -0,0 +1,16 @@
+{
+  "packages": [
+    {
+      "package": "go",
+      "version": "1.21.5",
+      "url": "https://dl.google.com/go/go1.21.5.linux-amd64.tar.gz",
+      "sha256": "e2bc0b3e4b64111ec117295c088bde5f00eeed1567999ff77bc859d7df70078e"
```

Comment:
nit: Do we control the `package` key? I think maybe `name` would better suited for this schema. That way we avoid having `package` name repeated multiple times: 

```go
type Package struct {
	Package string `json:"package"`
	Version string `json:"version"`
	SHA256  string `json:"sha256"`
	URL     string `json:"url"`
}
```

Reply:
Yep that's fine.